app/main/controller/audio_similarity_controller.py

```diff
@@ -17,7 +17,7 @@ class AudioSimilarityResource(Resource):
     @api.doc('Delete an audio in the similarity database')
     @api.expect(audio_similarity_request, validate=True)
     def delete(self):
-        return similarity.delete_item(request.args, "audio")
+        return similarity.delete_item(request.json, "audio") # Changed Since 4126 PR
```

Comment:
Are there any cases where we use `args` for delete in Check? I don't think so, but let's just double check. Same for Timpani @skyemeedan :pray: